scripts/publish-schema.sh

```diff
@@ -7,4 +7,5 @@ mkdir public
 # copy from src to preserve the structure
 cd src || exit 10
 find . -name schema.json -exec rsync -R {} ../public \; # https://stackoverflow.com/questions/11246070/cp-parents-option-on-mac/13855290#13855290
```

Comment:
I would suggest to actually publish the example as well, so let's remove the new added line and change this one to match any json files